.circleci/config.yml

```diff
@@ -105,11 +105,7 @@ references:
       environment:
         PACKAGES_DIR: *packages_workspace
       command: |
-        if [ "$(uname -s)" == "Darwin" ]; then
-            PKG_SUFFIX="osx-$(sw_vers -productVersion)"
```

Comment:
Please confirm this section is up-to-date https://github.com/aeternity/epoch/blob/c88ecb8a13cf7f160a8d5fe80bd53fed08539c33/docs/installation.md#retrieve-release-binary

Reply:
I don't see something contradicting with that section of the docs.